### PR TITLE
Nigeria Health Data - Number of Births

### PR DESCRIPTION
**Pull request by tebajanga**
## Description

Adding Number of Births health data for Nigeria. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
number_of_births

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
There are issues with the data as shown below:

screenshot from 2018-09-01 12-25-25

 + Negative percentage. Double check the calculations or is it possible to get HURUmap to calculate it?
 + The pie chart talks about **size**, any way to define this (like how 2.5kg was used in weights).

File: takwimu/profiles/census.py [hunk 1]
```diff
@@ -242,3 +243,49 @@ def get_causes_of_death(geo, session):
         'outpatient_diagnosis_over_five_dist': outpatient_diagnosis_over_five_dist,
         'outpatient_diagnosis_under_five_dist': outpatient_diagnosis_under_five_dist,
     }
+
+def get_number_of_births(geo, session):
```
Comment (kilemensi) [inline takwimu/profiles/census.py]: Lets just call these **Child births** i.e. `get_child_births`

File: takwimu/sql/less_weight.sql [hunk 2]
```diff
@@ -0,0 +1,91 @@
+--
```
Comment (kilemensi) [inline takwimu/sql/less_weight.sql]: May be **Child births with Low Birth Weights** ?

File: takwimu/sql/number_of_births.sql [hunk 3]
```diff
@@ -0,0 +1,96 @@
+--
```
Comment (kilemensi) [inline takwimu/sql/number_of_births.sql]: **Child births**

File: takwimu/sql/percentage_of_births_distribution.sql [hunk 4]
```diff
@@ -0,0 +1,210 @@
+--
```
Comment (kilemensi) [inline takwimu/sql/percentage_of_births_distribution.sql]: May be **Child births by size**?

File: takwimu/sql/reported_birth_weight.sql [hunk 5]
```diff
@@ -0,0 +1,96 @@
+--
```
Comment (kilemensi) [inline takwimu/sql/reported_birth_weight.sql]: **Child births with reported weights*?

**Comment by tebajanga** [comment]
I made the changes on the files. @kilemensi

**Pull request merged**